src/aks-preview/HISTORY.rst

```diff
@@ -13,6 +13,13 @@ Pending
 +++++++
 * Vendor new SDK and bump API version to 2023-11-02-preview.
 
+1.0.0b6
++++++++
+* Add `--ssh-access` to the `az aks create` command.
+* Add `--ssh-access` to the `az aks update` command.
+* Add `--ssh-access` to the `az aks nodepool add` command.
+* Add `--ssh-access` to the `az aks nodepool update` command.
```

Comment:
Could you please move `* Vendor new SDK and bump API version to 2023-11-02-preview.` from `Pending` to `1.0.0b6` too?

Reply:
updated.